hack/make-rules/test-cmd-util.sh

```
@@ -76,6 +76,43 @@ daemonsets="daemonsets"
 controllerrevisions="controllerrevisions"
 
 
+# include shell2junit library
+sh2ju="${KUBE_ROOT}/third_party/forked/shell2junit/sh2ju.sh"
+if [[ -f "${sh2ju}" ]]; then
+  source "${sh2ju}"
+else
+  echo "fetching sh2ju.sh from GitHub"
+  source <(curl -fsS --retry 3 'https://raw.githubusercontent.com/kubernetes/kubernetes/master/third_party/forked/shell2junit/sh2ju.sh')
+fi
```

Lets throw an error for now, and see if it passes the merge queue